### ISSUE TITLE
Relationship to Trace Compass?

### ISSUE THREAD

**Issue by jpouellet** [lttng/lttng-scope#21]
What is the relationship between this project and [Trace Compass](http://tracecompass.org/)?

I see the main committer here used to be a maintainer there? Is this a separate effort? If so, are there some docs someplace explaining the different goals?

**Comment by alexmonthy** [comment]
Hi,

Thanks for the interest! :)

LTTng-Scope is a separate effort. It is based on parts of the Trace Compass code, but it has different design goals, namely:
* Focus on CTF/LTTng use cases
* Streamline the user experience (few clicks to go from a trace to the relevant information)
* Separation of layers, for instance being able to run analyses independently of any UI toolkit like Eclipse

There are still a few things that need to be taken care of, including branding, before we can announce the project (more) publicly.

You do bring a good point, the README could mention these design goals. I will take care of that.
